lib/msal-common/src/authority/Authority.ts

```diff
@@ -444,8 +444,8 @@ export class Authority {
      */
     private async updateCloudDiscoveryMetadata(metadataEntity: AuthorityMetadataEntity): Promise<AuthorityMetadataSource> {
         this.logger.verbose("Attempting to get cloud discovery metadata in the config");
-        this.logger.verbosePii(`Known Authorities: ${this.authorityOptions.knownAuthorities || Constants.NOT_APPLICABLE}`);
-        this.logger.verbosePii(`Authority Metadata: ${this.authorityOptions.authorityMetadata || Constants.NOT_APPLICABLE}`);
+        this.logger.verbose(`Known Authorities: ${this.authorityOptions.knownAuthorities || Constants.NOT_APPLICABLE}`);
```

Comment:
PII

Reply:
@bgavrilMS and I had come to the conclusion that knownAuthorities and authorityMetadata aren't PII, and that only canonical_authority is.

Reply:
Both could contain tenant information which we've been treating as PII

Reply:
@tnorling - I think there is a naming confusion here. "known authorities" are just hosts, e.g. "login.microsoft.us", they do not have tenants/

Reply:
B2C authorities, which is the specific use case knownAuthorities is for, could include company names. In any case, it's supposed to be hosts only but there's nothing stopping people from providing the full authority (in fact I've seen people do this). We sanitize the provided values and pull out the hosts before using them but the raw provided value used here might still include data customers don't want to log.

Reply:
My general opinion on PII is that if we didn't create the data ourselves (user provided, server provided, etc) we should assume it contains PII

Reply:
There is an Enterprise data taxonomy that helps clarify this officially. I am sharing it offline and please make a decision accordingly. My personal take is to err on the side of caution if we are doubtful reg classification.